configuration.example.yaml

```diff
@@ -1,55 +0,0 @@
-# Backend
```

Comment:
If you review the workflow of the client library, if there are issues with **authorization**. that is solved in the very first process of authentication, if there are 403 error there, the workflow is disrupted and continues only if the user has correct *authorization**
In addition, remember we developed the *user roles* endpoint in order to solve **authorization** in the very first stage of the workflow
If you review, it's unnecessary add 403 exceptions due to using roles approach (that is actually how it works)

Reply:
I did not say that the problem was on auth, i said that there are 403 responses after that, with the other funcionalities, and these are not managed like you did in auth. For example if the user is an admin and try to do a submission, there is a 403 on the server but in the client is not displayed like a 403, because is not caught

Reply:
Ok but the problem isn't here

Reply:
Well, you can tell, if the problem is here, or in the client lib, because you are sending responses that are not managed correctly in the flow, and because in the client lib there was not PR im telling you upfront

Reply:
Ok, I change the server-side authorization approach